### DISCUSSION TITLE
Offer search field for options

### DISCUSSION THREAD

**Discussion by saschafoerster** [pretix/pretix#2740] [Ideas]
As Pretix has many options, sometimes I don't know in which submenu an option is hidden. Like today I was searching for an option to allow customers to switch products themself after the ordered. If I don't know the system to well, i have to check many pages. So I was searching for a search field that helps finding the right way to change a certain option field. 

RocketChat allows such a search in it's administrator backend. It doesn't only search the title, but also all the fields behind it and guides my by only leaving the menu items, that might be a path to the right option. 

This is a screenshot of RocketChat Admin backend as an example. 
<img width="890" alt="Bildschirmfoto 2022-07-28 um 20 53 53" src="https://user-images.githubusercontent.com/1153741/181616281-b09de9f9-a728-4c09-af69-4564b53680bd.png">

**Comment by raphaelm** [comment]
Yeah, I like the idea. How realistic it is to build probably depends on the scope (event-level settings only? or including things like product settings? including plugins?).

A downside of this is of course that you need to know roughly how the option is phrased, so it's no replacement for good structure, but it might be a helpful addition.

**Reply by saschafoerster** [reply]
I think all options are a good start. Product details would be a nice addition, but product options are essential (not the content, just the name of the option). Plug-in name would be enough for me as well.

**Reply by raphaelm** [reply]
what's do you mean by "product details vs product options", what is the difference there?

I was referring to "everything on the menu named 'settings'" vs "everything in every menu of pretix"

**Reply by saschafoerster** [reply]
Okay, I missunderstood between "name of the setting" vs. "content of the fields of certain settings" like the product settings content, which is individual, while the name of a setting is constant. 

For a starter, I would be happy for a search of all the settings of one certain event.
